lib/vf_analyzers.cpp

```diff
@@ -941,8 +941,11 @@ struct MultiValueFlowAnalyzer : ValueFlowAnalyzer {
     MultiValueFlowAnalyzer(const std::unordered_map<const Variable*, ValueFlow::Value>& args, const Settings& set)
         : ValueFlowAnalyzer(set) {
         for (const auto& p:args) {
-            values[p.first->declarationId()] = p.second;
-            vars[p.first->declarationId()] = p.first;
+            const auto declId = p.first->declarationId();
+            if (declId == 0)
+                continue; // TODO: should never happen?
```

Comment:
This occurs in e.g. `TestAutoVariables::testinvaliddealloc`.

The `Variable` has no `mNameToken` and points to `void*` which appears to be the injected parameter for `free()`. The `errorPath` in the values seems to support this:
```
Address of variable taken here.
```
```
Calling function 'free', 1st argument '&c2' value is lifetime=c2
```

Not sure if something with the injecting is lacking data. But I do not think this needs to be solved in the scope of this PR.

@pfultz2 Any thoughts?